bridge/client/app/_models/project.ts

```diff
@@ -24,6 +25,16 @@ export class Project {
     return this.services;
   }
 
+  getShipyardVersion(): string {
+    return this.shipyardVersion?.split('/').pop();
+  }
+
+  isShipyardNotSupported(): boolean {
+    const supported = '0.2.0'.split('.');
+    const current = this.getShipyardVersion()?.split('.');
+    return current && +current[0] <= +supported[0] && +current[1] < +supported[1];
+  }
```

Comment:
I think this is communicated differently in #2909, the spec version (0.2.0) should be provided by the metadata endpoint of the keptn api

Reply:
@ermin-muratovic do we have a library for comparing (semantic) versions (instead of manually doing the comparison)?

Reply:
Yes, we have a library. My bad